<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LaunchWindow</class>
 <widget class="QMainWindow" name="LaunchWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>743</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>RCL Creator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>1000</width>
     <height>700</height>
    </size>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>1000</width>
        <height>700</height>
       </size>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="tabPosition">
       <enum>QTabWidget::TabPosition::West</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="rcl::gui::resume::ResumeTabWidget" name="resumeTab">
       <attribute name="icon">
        <iconset theme="QIcon::ThemeIcon::DocumentNew"/>
       </attribute>
       <attribute name="title">
        <string>Resume</string>
       </attribute>
      </widget>
      <widget class="rcl::gui::cl::CoverLetterTab" name="coverLetterTab">
       <attribute name="icon">
        <iconset theme="QIcon::ThemeIcon::DocumentNew"/>
       </attribute>
       <attribute name="title">
        <string>Cover Letter</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSave"/>
    <addaction name="actionKeywords"/>
   </widget>
   <widget class="QMenu" name="menuMenu_2">
    <property name="title">
     <string>Jobs</string>
    </property>
    <addaction name="actionImportJob"/>
   </widget>
   <addaction name="menuMenu"/>
   <addaction name="menuMenu_2"/>
  </widget>
  <action name="actionImportJob">
   <property name="text">
    <string>Import Specification</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionKeywords">
   <property name="text">
    <string>Keywords</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>rcl::gui::resume::ResumeTabWidget</class>
   <extends>QWidget</extends>
   <header>resumetabwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>rcl::gui::cl::CoverLetterTab</class>
   <extends>QWidget</extends>
   <header>coverlettertab.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
